modules/features/falcon_feature_thankq/config/install/core.entity_form_display.source_code.source_code.default.yml

```diff
@@ -0,0 +1,38 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - eck.eck_type.source_code.source_code
+    - field.field.source_code.source_code.field_availability
+    - field.field.source_code.source_code.field_label
+    - field.field.source_code.source_code.field_source_code
+  module:
+    - datetime_range
+id: source_code.source_code.default
+targetEntityType: source_code
+bundle: source_code
+mode: default
+content:
+  field_availability:
+    weight: 5
+    settings: {  }
+    third_party_settings: {  }
+    type: daterange_default
+    region: content
+  field_label:
+    weight: 4
+    settings:
+      size: 30
+      placeholder: 'I saw an ad on TV'
+    third_party_settings: {  }
+    type: string_textfield
+    region: content
+  field_source_code:
+    weight: 3
+    settings:
+      size: 10
+      placeholder: ASDR18-10
```

Comment:
I'd avoid real values in placeholders. For me **personally**, it's a bit confusing. I'd either remove the placeholder or make it something like "XXXX00-00"